packages/storybook/stories/toggletip.mdx

```diff
@@ -0,0 +1,64 @@
+import { Toggletip } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as ToggletipStories  from './toggletip.stories';
+
+<Meta of={ToggletipStories} />
+
+# Toggletip
+
+## Table of contents
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Toggletips are used to display additional information about a target element, without the user losing the context of
+the page [2].
+<Canvas of={ToggletipStories.Default} />
+
+## Usage
+
+### When to use
+- Use sparingly and only when the information cannot be presented to the user in any other way (e.g. directly on the
+page)[7].
+- Provides additional information about a page element (feature, action, content, etc) [1].
```

Comment:
Question de garder la même formulation que les autres points. Commence la phrase avec "Use to..."
> Use to provide additional...